src/Core/Log.re

```diff
@@ -106,19 +101,26 @@ module Namespace = {
 
 type msgf('a, 'b) = (format4('a, Format.formatter, unit, 'b) => 'a) => 'b;
 
+let logFileChannel = ref(None);
+
+let setLogFile = path => {
+  Option.iter(close_out, logFileChannel^);
+  let channel = open_out(path);
+  Printf.fprintf(channel, "Starting log file.\n%!");
+  logFileChannel := Some(channel);
+};
```

Comment:
Should this also always set the log level to debug, as it does when the env variable is set?

Reply:
I think keeping them as separate options is OK 👍